src/components/Capture/ProfileData.tsx

```diff
@@ -291,12 +292,10 @@ const getFieldComponent = (
       return <Input {...props} type="text" style={{ width: space(22) }} />
     case 'ssn':
       return (
-        <Input
+        <SSNInput
           {...props}
           placeholder={'999-99-9999'}
-          pattern={'d{3}-d{2}-d{4}'}
           maxLength={11}
```

Comment:
IMO better to put `maxLength` inside custom component also? And probably `placeholder` too?